components/FilterBar/components/Content/components/FilterButton/connector.js

```diff
@@ -1,13 +1,13 @@
 import { connect } from 'react-redux';
-import openFilterView from '../../actions/openFilterView';
+import openFilterRoute from '../../actions/openFilterRoute';
 
 /**
  * @param {Function} dispatch The redux dispatch function.
  * @param {Object} props The components props.
  * @return {Object}
  */
 const mapDispatchToProps = dispatch => ({
-  navigate: () => dispatch(openFilterView()),
+  navigate: () => dispatch(openFilterRoute()),
 });
 
 export default connect(null, mapDispatchToProps, null, { pure: () => null });
```

Comment:
We introduced redux-props. Why not refactor this here a little to use it?

Reply:
Will be done in another ticket.